index.html

```diff
@@ -200,5 +200,37 @@ <h1>About</h1>
                         </div>
                     </div>
                 </section>
+
+                <section class="contactForm">
+                    <div class="contactLogo"><img src="./images/logo.png"
+                            alt="Logo"></div>
+                    <div class="content">
+                        <h1>Get started</h1>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing
+                            elit, sed do eiusmod tempor incididunt ut labore et
+                            dolore magna.</p>
+                    </div>
+                    <form action="https://formspree.io/f/mjvzkrzj"  method="POST">
+                        <input type="text" required id="fullName" placeholder="Your name"
+                            name="Full Name" maxlength="30">
+                        <input type="email" required="required" name="" Email id="myEmail"
+                            placeholder="Your email">
+                        <textarea name="subject" id="msg" placeholder="Enter your message here..." maxlength="500" required></textarea>
+                        <input type="submit" value="Start collaboration" id="button">
+                    </form>
+                </section>
+                <section class="myFooter">
```

Comment:
- For a better HTML structure, I advise you do not need to wrap your `footer` tag in a `section` tag. I suggest you remove the `section` tag.

Reply:
_Thanks. I will work on that._